src/replicache.ts

```diff
@@ -97,7 +97,7 @@ type StorageBroadcastData = Omit<BroadcastData, 'changedKeys'> & {
 };
 
 /**
- * The type used to describe the mutator definitions passed into [[Replicache]]
+ * The type used to describe the mutator definitions passed into [Replicache](classes/Replicache)
```

Comment:
This is a workaround... Without this it outputs the following markdown:

```md
[Replicache]()
```

I guess it could be because of dependency ordering?